weather-forecast/ecrad-3d-correction/unets/README.md

```diff
@@ -2,24 +2,84 @@
+
+In the atmosphere, absorption, emission or scattering of short-wave (SW) and long-wave (LW) radiation are important processes. Short-wave radiation is the major source of energy of the Earth system. Long-wave radiation is emitted by the Earth surface and the atmosphere and is responsible for the greenhouse effect.
+
+The solution of the radiative transfer equations to obtain the fluxes are computationnaly expensive and, in practice, calculations are performed on a coarser resolution and/or using an increased timestep (lower time frequency). The fluxes are later interpolated back to the original (finer) grid. The Integrated Forecasting System (IFS), the operational numerical weather prediction model developed by ECMWF, comes with the ecRad scheme (https://github.com/ecmwf-ifs/ecrad) that implements several solvers. Two of them represent the horizontal cloud inhomogeneity: Tripleclouds *(Shonk and Hogan, 2008)* and SPARTACUS ( Speedy Algorithm for Radiative Transfer through Cloud Sides; *Hogan et al., 2016*). SPARTACUS can be seen an extension of Tripleclouds by treating the 3D radiative effects associated with clouds but is currently too expensive for operational weather predictions (approximatively five times slower).\
+*N.B.: This number should be updated based on Ukkonen and Hogan (2024) but one should keep in mind that they reduced the precision of the two-stream kernel computations to achieve better performance.*
+
+The present task aims at learning the 3D cloud radiative effects that will be integrated to Tripleclouds formulation as a corrective term. Learning a corrective term instead of the entire SPARTACUS solver focuses on the most computationally intensive aspects of this solver.
+The neural network has to learn the difference between the outputs of SPARTACUS and Tripleclouds (SW/LW fluxes and heating rates).
+
+**This work builds on the research of *Meyer et al. (2021)* but uses higher-quality data and a more suitable neural network.**
 
 ## Dataset
 
-Download the dataset using the Climetlab library developed by ECMWF. An example on how to retrieve the data is largely explained [here](https://git.ecmwf.int/projects/MLFET/repos/maelstrom-radiation/browse).
+The dataset has been generated by ECMWF as a delivrable of the [MAELSTROM project](https://www.maelstrom-eurohpc.eu/) project. It can be downloaded using the [Climetlab](https://github.com/ecmwf/climetlab) library also developed by ECMWF.
 
-## Models
+The dataset is widely explained in the MAESLTROM radiation repository (https://git.ecmwf.int/projects/MLFET/repos/maelstrom-radiation/browse). 
+
+Before training the neural network, a preprocessing step is required that will transform the NetCDF dataset into a Zarr dataset.
+The parameters for the normalization (means and standard deviations) are computed during this step. 
 
-X:
+Original features:
 
-- ??
+* ``sca_inputs`` (*column x sca_variable*):
+    - skin_temperature,
```

Comment:
Can we find somewhere (maybe in ECMWF references) the brief description of each variable ?
Otherwise, could it be useful, for non-expert, to add some words next to each variable name.

Reply:
I added the link to ecRad User Guide for a description.